### PR TITLE
Add fields & commands for Character Export

### PR DESCRIPTION
**Pull request by EJWellman**

### CODE REVIEW

File: zone/client.cpp [hunk 1]
```diff
@@ -6912,6 +6912,34 @@ bool Client::IsMarried()
...
 
+void Client::SetCharExportFlag(uint8 flag)
+{
+	if (flag == 0) {
+		m_epp.char_export_flag = 0;
+		Save(1);
+		Message(CC_Default, "Character export disabled.");
+		return;
+	}
+	if (flag == 1) {
```
Comment (SecretsOTheP) [inline zone/client.cpp:6923]: Change this to else if for optimization purposes.

File: zone/client.h [hunk 2]
```diff
@@ -380,6 +380,10 @@ class Client : public Mob
 	void SetMarried(const char* playerName);
 	bool IsMarried();
 
+	uint8 GetCharExportFlag() { return m_epp.char_export_flag; };
```
Comment (SecretsOTheP) [inline zone/client.h:385]: Nit: inline uint8

File: zone/command.cpp [hunk 3]
```diff
@@ -9016,6 +9017,50 @@ void command_reloadtitles(Client *c, const Seperator *sep){
...
+			c->Message(CC_Default, "Export is currently off.");
+		else if (flagval == 1)
+			c->Message(CC_Default, "Export is currently set to include character info and worn items.");
+		else if (flagval == 2)
+			c->Message(CC_Default, "Export is currently set to include character info, worn items and inventory.");
+		else if (flagval == 3)
+			c->Message(CC_Default, "Export is currently set to include character info, worn items, inventory, and bank.");
+		return;
+	}
+	if (!strcasecmp(sep->arg[1], "off")) {
```
Comment (SecretsOTheP) [inline zone/command.cpp:9035]: if / elseif - optimization.

File: zone/command.cpp [hunk 4]
```diff
@@ -9016,6 +9017,50 @@ void command_reloadtitles(Client *c, const Seperator *sep){
...
+	if (!strcasecmp(sep->arg[1], "inventory")) {
+		c->SetCharExportFlag(2);
+		return;
+	}
+	if (!strcasecmp(sep->arg[1], "bank")) {
+		c->SetCharExportFlag(3);
+		return;
+	}
+
+	if (arguments) { c->Message(CC_Default, "Invalid argument"); }
```
Comment (SecretsOTheP) [inline zone/command.cpp:9052]: Nit: please keep this consistent with what I've been using ie; 
(preferred)

```
if (arguments) 
{ 
c->Message(CC_Default, "Invalid argument");
 }
```

or, un preferred
```
if (arguments) { 
c->Message(CC_Default, "Invalid argument");
 }
```
Makes it cleaner to read.

File: zone/zonedb.cpp [hunk 5]
```diff
@@ -1288,7 +1291,8 @@ bool ZoneDatabase::SaveCharacterData(uint32 character_id, uint32 account_id, Pla
 		"%u,"   // e_betabuff_gear_flag
 		"%lu, "   // e_zone_guild_id
 		"'%s', "  // e_temp_last_name
-		"%u "  // e_married_character_id
+		"%u, "  // e_married_character_id
```
Comment (SecretsOTheP) [inline zone/zonedb.cpp:1294]: Nit; extra space.

**Review by SecretsOTheP — CHANGES_REQUESTED** [review]
There's a few nits, don't need to fix those.
But please change the if/else if chain :)

File: zone/client.cpp (hunk 1 above)
Comment (EJWellman) [reply zone/client.cpp]: Updated to if-else if chain

File: zone/zonedb.cpp (hunk 5 above)
Comment (EJWellman) [reply zone/zonedb.cpp]: Cleaned up the next couple up, as well.

**Review by SecretsOTheP — APPROVED** [review]

**Pull request merged**